docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md

```diff
@@ -1,8 +1,166 @@
 
 # Restaking Developer Guide
 
-Direct smart contract Restaking allows users to interact with the public smart contracts for a more customized Restaking experience. Please find the following developer resources to get started:
+Smart Contract Restaking allows the user to interact directly with the EigenLayer core contracts. The following sections describe how to setup your Restaking integration with the EigenLayer contracts directly with no reliance on the EigenLayer Web App.
+
+The Smart Contract Restaking user guides below serve as a high level overview of the most important steps for required for Restaking.
+
+Please note the following resources are available for users that wish to take a deeper dive on the contracts:
+* [EigenLayer Protocol Developer Docs](https://github.com/Layr-Labs/eigenlayer-contracts/tree/dev/docs): detailed protocol specifications for restaking smart contract integration developers.
+* [EigenLayer Protocol Contract Addresses](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#deployments): deployed contract addresses for Mainnet and Testnet.
+* [EigenLayer Protocol Integration Tests](https://github.com/Layr-Labs/eigenlayer-contracts/tree/dev/src/test/integration): tests that serve as examples on how to interact with the EigenLayer core contracts.
+
+
+
+
+
+# Smart Contract Liquid Restaking User Guide
+
+## Steps to Deposit (Restake) Liquid Tokens
+
+1. Call StrategyManager.depositIntoStrategy() .
+2. User is now actively Restaked.
+
+## Steps to Withdraw (Unstake) Liquid Tokens
+
+1. Queue Withdrawal: call DelegationManager.queueWithdrawal() to trigger the escrow period. Wait for Escrow Period: 24 days for $EIGEN, 7 days for all other tokens. Please see further detail [here](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/#escrow-period-withdrawal-delay).
+
+2. Complete Withdrawal as Tokens: call DelegationManager.completeQueuedWithdrawal() to complete the withdrawal and return assets to the withdrawer's wallet.
+
+
+# Smart Contract Delegation User Guide
+
+The process of Delegating assets is the same for both liquid and native restaked assets. The user's Restaking wallet must Delegate all restaked assets to a single Operator. After the initial Delegate operation - any subsequent Deposited (Restaked) assets are also automatically delegated to the current operator.
+
+## Steps to Delegate Assets
+
+1. Call DelegationManager.delegateTo()
```

Comment:
Might be helpful to note that they need to pass in the address of the operator they want to delegate to. There are 3 parameters:
* `operator`: the address of the operator you want to delegate to
* `approverSignatureAndExpiry`: can be left blank
* `approverSalt`: can be left blank